src/main/java/org/sagebionetworks/web/client/presenter/SettingsPresenter.java

```diff
@@ -255,6 +260,32 @@ public void configure() {
     view.setOauthClientSettingsVisible(
       DisplayUtils.isInTestWebsite(ginInjector.getCookieProvider())
     );
+
+    if (DisplayUtils.isInTestWebsite(ginInjector.getCookieProvider())) {
+      view.setTwoFactorAuthSettingsVisible(true);
+    } else {
+      // If 2FA is enabled, show the settings pane, even if experimental mode is disabled
```

Comment:
Small block to guard against the case where someone adds 2FA but then can't log into apps that haven't been updated.